mgmtsystem_survey/readme/CREDITS.md

```diff
@@ -0,0 +1,3 @@
+The development of this module has been financially supported by:
+
+- Camptocamp
```

Comment:
missing this content: `The migration of this module from 17.0 to 18.0 was financially supported by Camptocamp.
`